include/common/utils/algorithm.h

```diff
@@ -32,7 +32,8 @@ template <ranges::range Range>
 auto
 getUniqueElements( Range&& range )
 {
-    return std::forward<Range>( range ) | ranges::views::all | ranges::views::unique | ranges::to_vector;
+    auto sorted = std::forward<Range>( range ) | ranges::to_vector | ranges::actions::sort;
+    return ranges::actions::unique( sorted );
```

Comment:
Why does this sort the data?

Reply:
From Range-v3 documentation:
"Given a range, return a new range where all consecutive elements that compare equal save the first have been filtered out. "
Without sorting we could get duplicated elements.